R/wb_functions.R

```diff
@@ -639,6 +635,30 @@ wb_to_df <- function(
 
   }
 
+  # is.na needs convert
+  if (skipEmptyRows) {
+    empty <- apply(z, 1, function(x) all(is.na(x)), simplify = TRUE)
```

Comment:
`simplify` was added to `apply()` in **R 4.1.0**; current dependency is `R (>= 3.4.0)`.

Reply:
Oh woops, thanks. It's in there for a long long time.
Otherwise any gut feeling whether or not we should fulfill this feature request?

Reply:
Keeping named `NA` columns seems better than accidentally dropping them.  Users can always drop the columns afterwards.